src/styles/cards.scss

```diff
@@ -0,0 +1,118 @@
+@import "./variables.scss";
+
+body,
+h2,
+p,
+.page {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 4 * $card-width + 3 * $card-margin;
+  padding-top: 50px;
+  margin: auto;
+}
```

Comment:
this does not apply to the card block

Reply:
and don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.